src/org/oidc/common/UnsupportedContentType.java

```diff
@@ -0,0 +1,7 @@
+package org.oidc.common;
+
+public class UnsupportedContentType extends Exception{
```

Comment:
Every custom exception should end with Exception. Rename it as UnsupportedContentTypeException